analysis/plotconfigs/svt/rawSvtAnaHits2.json

```diff
@@ -0,0 +1,812 @@
+{   
```

Comment:
Why are there 3 different version of this with just a difference of a number in the filename?

Reply:
I have left the non obsolete one. Renamed it without the numbering. The numbers referred to different versions with different diagnostic plots. It seems like they were just different bounds on the plots; I haven't used earlier ones in eons.